core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java

```diff
@@ -246,6 +246,7 @@ private void ifFlowNotRunning(Runnable function) {
         return EnumSet.of(
```

Comment:
**suggestion (non-blocking):** This isn't part of your change, but the misspelling of this method name feels insidious. Consider refactoring it. 

```diff
diff --git a/core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java b/core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java
index 0c98b0e98e..6ec2104928 100644
--- a/core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java
+++ b/core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java
@@ -245 +245 @@ public class StackStatusCheckerJob extends StatusCheckerJob {
-    private EnumSet<Status> statesFromAvailabeAllowed() {
+    private EnumSet<Status> statesFromAvailableAllowed() {
```

Reply:
You are right and have a higher standard than me. HAHAHA . Corrected those now.